float64 initial_start
float64 initial_goal

string name

dynamic_movement_primitive/Model lwr_model
